Explorer/Assets/DCL/Multiplayer/Connections/Messaging/Pipe/MessagePipe.cs

```diff
@@ -0,0 +1,115 @@
+using DCL.Utilities.Extensions;
+using Decentraland.Kernel.Comms.Rfc4;
+using Google.Protobuf;
+using LiveKit.Internal.FFIClients.Pools;
+using LiveKit.Internal.FFIClients.Pools.Memory;
+using LiveKit.Proto;
+using LiveKit.Rooms.DataPipes;
+using LiveKit.Rooms.Participants;
+using System;
+using System.Collections.Generic;
+
+namespace DCL.Multiplayer.Connections.Messaging.Pipe
+{
+    public class MessagePipe : IMessagePipe
+    {
+        private readonly IDataPipe dataPipe;
+        private readonly IMultiPool multiPool;
+        private readonly IMemoryPool memoryPool;
+        private readonly MessageParser<Packet> messageParser;
+
+        private readonly Dictionary<string, List<Action<(Packet, Participant)>>> subscribers = new ();
+
+        private static readonly ISet<string> SUPPORTED_TYPES = new HashSet<string>
+        {
+            nameof(Position),
```

Comment:
If it is about the message with Position, Rotation and Animation, then I would rather call it `ProfileState` or `AvatarState`, or at least `Transfrom`

Reply:
It's about message for Decentraland.Kernel.Comms.Rfc4